Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyFilePathCtrl.h

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#if !defined(Q_MOC_RUN)
+#include <AzCore/IO/Path/Path.h>
+#include <AzCore/Memory/SystemAllocator.h>
+
+#include <QString>
+#include <QWidget>
+
+#include "PropertyEditorAPI.h"
+#endif
+
+
+namespace AzQtComponents
+{
+    class BrowseEdit;
+}
+
+namespace AzToolsFramework
+{
+    // A control for representing an AZ::IO::Path field with
+    // a BrowseEdit whose popup button triggers a native
+    // file dialog
```

Comment:
nit: should be `//!` style (and missing `.`)

Reply:
Thanks will fix!